src/main/java/org/dependencytrack/model/ConfigPropertyConstants.java

```diff
@@ -79,8 +79,8 @@ public enum ConfigPropertyConstants {
     KENNA_TOKEN("integrations", "kenna.token", null, PropertyType.ENCRYPTEDSTRING, "The token to use when authenticating to Kenna Security"),
     KENNA_CONNECTOR_ID("integrations", "kenna.connector.id", null, PropertyType.STRING, "The Kenna Security connector identifier to upload to"),
     ACCESS_MANAGEMENT_ACL_ENABLED("access-management", "acl.enabled", "false", PropertyType.BOOLEAN, "Flag to enable/disable access control to projects in the portfolio"),
-    NOTIFICATION_TEMPLATE_BASE_DIR("notification", "template.baseDir", SystemUtils.getEnvironmentVariable("DEFAULT_TEMPLATES_OVERRIDE_BASE_DIRECTORY", System.getProperty("user.home")), PropertyType.STRING, "The base directory to use when searching for notification templates"),
-    NOTIFICATION_TEMPLATE_DEFAULT_OVERRIDE_ENABLED("notification", "template.default.override.enabled", SystemUtils.getEnvironmentVariable("DEFAULT_TEMPLATES_OVERRIDE_ENABLED", "false"), PropertyType.BOOLEAN, "Flag to enable/disable override of default notification templates"),
+    NOTIFICATION_TEMPLATE_BASE_DIR("notification", "template.baseDir", SystemUtils.getEnvironmentVariable("DT_DEFAULT_TEMPLATES_OVERRIDE_BASE_DIRECTORY", SystemUtils.getEnvironmentVariable("DEFAULT_TEMPLATES_OVERRIDE_BASE_DIRECTORY", System.getProperty("user.home"))), PropertyType.STRING, "The base directory to use when searching for notification templates"),
```

Comment:
@syalioune maybe im misinterpreting this line.  in the deploy-docker.md and docker-compose.yml you replaced:

`DEFAULT_TEMPLATES_OVERRIDE_BASE_DIRECTORY`
with
`DT_DEFAULT_TEMPLATES_OVERRIDE_BASE_DIRECTORY`

should this state: 
....`SystemUtils.getEnvironmentVariable("DT_DEFAULT_TEMPLATES_OVERRIDE_BASE_DIRECTORY", System.get.`....

Reply:
similar comment for the next line "`DEFAULT_TEMPLATES_OVERRIDE_ENABLED`" is called, but was removed earlier.

Reply:
Hello @melba-lopez 
Sorry I missed your comment.
The intent is to be backward compatible with previous releases where `DEFAULT_TEMPLATES_OVERRIDE_.*` variables were valid so the code fetches the values from (decreasing priority order) :

- `DT_DEFAULT_TEMPLATES_.*`
- `DEFAULT_TEMPLATES_.*`
- `user.home`